### PR TITLE
feat: docs: updated readme files for better understanding and development

### PR DESCRIPTION
**Pull request by 0xCipherCoder**
Changes - 
 
 - APT-SETUP.md
  • Comprehensive package management and installation guide
  • System setup and troubleshooting steps

- DEVELEOPMENT.md 
  • Detailed development workflow and project architecture
  • Build process, testing guidelines and contribution steps

-  README.md
  • Clearer project overview and feature documentation
  • Improved structure and quick-start guides

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,35 +2,52 @@
...
 All of these features are crafted to reduce time to market and let you focus on your core products, rather than the complexity of maintaining and operating the network.
 
 If you have questions or want to contribute, join us in the SVM community chat on Telegram or make a pull request.
 
-## Next Steps:
+## Next Steps
 
 - [SVMKit Examples](https://github.com/abklabs/svmkit-examples) - A collection of examples on how to use SVMKit in your environment.
-- [Developer Information](DEV.md) - Interested in developing with SVMKit?  Take a look here.
+- [Developer Information](/docs/DEV.md) - Interested in developing with SVMKit? Take a look here.
```
[the remark this answers is not in the document]
Comment (0xCipherCoder) [reply README.md]: Yes updated it.

File: docs/APT-SETUP.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+- Multiple validator implementations (Agave, Solana, Jito, etc.)
+- CLI tools and utilities
+- Regular updates and security patches
+- Seamless integration with Ubuntu/Debian systems
+
+## Available Packages
+
+| Package Name                   | Description                          | Dependencies    |
+| ------------------------------ | ------------------------------------ | --------------- |
```
[the remark this answers is not in the document]
Comment (0xCipherCoder) [reply docs/APT-SETUP.md]: Sure removed the dependencies column.

File: docs/DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+~/go/bin/golangci-lint --version
+make --version
+```
+
+## Project Structure
+
+```sh
+svmkit/
+├── agave/                 # Agave validator implementation
+│   ├── assets/           # Validator deployment assets
```
[the remark this answers is not in the document]
Comment (0xCipherCoder) [reply docs/DEVELOPMENT.md]: I have updated the project structure to exclude individual files and aligned comments vertically.

**Comment by 0xCipherCoder** [comment]
> Great; thank you for the improvements. I left a couple of really minor notes. I can merge this in after.

Thanks for the review. @alexanderguy I have updated the readme files as per the comments. Please check.

**Review by alexanderguy — COMMENTED** [review]
LGTMl; thank you.  

We do rebasing for PR merges on this project.  Could you please squash your commits into 1 (since we don't need the update history) and I'll merge it in after that.

**Comment by 0xCipherCoder** [comment]
@alexanderguy I tried to rebase and squash but looks like mistakenly other changes also got added. I am not getting when those added.

**Comment by alexanderguy** [comment]
OK.  Those changes already in `main` so perhaps when you rebased, they got sucked in, and now they're squashed.  All you need to do is fetch from our repo, and rebase on our `main` again.  It should clean that out.

**Comment by 0xCipherCoder** [comment]
> OK. Those changes already in `main` so perhaps when you rebased, they got sucked in, and now they're squashed. All you need to do is fetch from our repo, and rebase on our `main` again. It should clean that out.

I'm sorry but when I tried to do this it messed up the commit history and resolving that again caused multiple conflicts resolution while rebase. What is best way now should I create new PR directly with final changes?

**Comment by alexanderguy** [comment]
No worries!  When I'm back at a machine I'll rebase and merge it manually.

**Comment by alexanderguy** [comment]
I merged this in manually.  Thanks for the PR!

**Pull request closed without merging**